version.gradle.kts

```diff
@@ -24,11 +24,11 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-val baseVersion: String by extra("2.0.0-SNAPSHOT.114")
-val coreVersion: String by extra("2.0.0-SNAPSHOT.116")
+val baseVersion: String by extra("2.0.0-SNAPSHOT.115")
+val coreVersion: String by extra("2.0.0-SNAPSHOT.117")
 val toolBaseVersion: String by extra("2.0.0-SNAPSHOT.111")
 val mcVersion: String by extra("2.0.0-SNAPSHOT.90")
-val mcJavaVersion: String by extra("2.0.0-SNAPSHOT.102")
+val mcJavaVersion: String by extra("2.0.0-SNAPSHOT.103")
 
 // The version of ProtoData being developed.
```

Comment:
I think it's more like the version to publish. We had this comment about "being developed" to distinguish this version from the one used to build these sources (`protoDataDevVersion`), which is no longer needed, as it is now embedded into `modelCompiler`, as far as I understand.

Reply:
Added issue: https://github.com/SpineEventEngine/config/issues/418